lib/tasks/cron.rake

```diff
@@ -15,6 +15,16 @@ namespace :cron do
     UserTotal.refresh_view
   end
 
+  desc "Send second slip notification"
+  task second_slip_notification: :environment do
+    puts "sending second slip notification"
+    ContributionDetail.where(payment_method: 'BoletoBancario',
```

Comment:
This ```where``` should be a scope in the model with an accompanying spec. Something like ```slips_past_waiting``` or ```slips_waiting_forever```